composer/trainer/trainer.py

```diff
@@ -681,12 +704,9 @@ class Trainer:
             ``'tpu'``, or ``'mps'``. (default: ``None``)
 
             The default behavior sets the device to ``'gpu'`` if CUDA is available, and otherwise ``'cpu'``.
-        precision (Precision | str, optional): Numerical precision to use for training. One of ``fp32``, ``fp16``
-            or ``amp`` (recommended). (default: ``Precision.FP32`` if training on CPU; ``Precision.AMP`` if training
-            on GPU)
-
-            .. note::
-                ``fp16`` only works if ``deepspeed_config`` is also provided.
+        precision (Precision | str, optional): Numerical precision to use for training. One of ``fp32``, ``amp_fp16``
+            or ``amp_bf16`` (recommended). (default: ``Precision.FP32`` if training on CPU; ``Precision.AMP_FP16`` if
```

Comment:
Why is amp_bf16 recommended? This is only true for certain classes of NLP models and may be slower than amp_fp16.

Reply:
Corrected